src/main/java/com/google/devtools/build/lib/analysis/config/ExecutionTransitionFactory.java

```diff
@@ -185,6 +185,7 @@ private static BuildOptions transitionImpl(BuildOptionsView options, Label execu
           break;
         default:
           // else if OFF do nothing
```

Comment:
Can you update this comment to "else if OFF just mark that we are now in an exec transition"

The mark shouldn't be necessary for correctness; however, it is very convenient for many classes of user to be able to look at the shortname and see that it is an exec configuration.

Reply:
Done.

I have seen rulesets check `ctx.bin_dir.path` for `-exec-`, so I would assume removing that entirely would negatively affect downstream tests. It's also useful for humans though, I agree.